lib/trento/vault.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Trento.Vault do
+  @moduledoc """
+  Configuration for Cloak Vault at startup
+  """
+
+  use Cloak.Vault, otp_app: :trento
+
+  @impl GenServer
+  def init(config) do
+    config =
+      Keyword.put(config, :ciphers,
+        default: {
+          Cloak.Ciphers.AES.GCM,
+          tag: "AES.GCM.V1",
+          key: Base.decode64!(Application.fetch_env!(:trento, Trento.Vault)[:encryption_key_base]),
+          iv_length: 12
+        }
+      )
+
+    {:ok, config}
+  end
```

Comment:
You can use just `use Cloak.Vault` avoiding this callback providing the configuration directly inside the config files.